docs/plugins/testing_workshop_docs/Test-coverage.md

```diff
@@ -0,0 +1,101 @@
+This article will be more valuable if you are familiar with the Python programming language and napari software. It is the fourth in a series of articles on testing taken from the [January 2022 testing workshop video](https://drive.google.com/file/d/1DaMrRz-rLRQ6-_y0J8O3GRpVPCn0rgYs/view). The information in this article starts at minute 28:26. This article is a summary of the information in the video and should stand on its own. The other articles are:  
+Article 1: [Python’s assert keyword](./Pythons-assert-keyword)  
+Article 2: [Pytest testing framework](./Pytest-testing-frameworks)  
+Article 3: [Readers and fixtures](./Readers-and-fixtures)  
+Article 4: This article  
+Article 5: Testing widgets  
+
+This article covers:   
+* [Coverage](#coverage)
+* [pytest --cov](#pytest---cov)    
+ 
+#### Resources  
+The example plugin and all the tests discussed in this article are available in [this GitHub repository](https://github.com/DragaDoncila/plugin-tests).    
+
+## Coverage  
+How do we know when we have tested everything? 
+
+This is where coverage comes in. `pytest-cov` can find out what the coverage of our tests is. Install using 
+`pip install pytest-cov`. This feature provides coverage data for pytest. Coverage is the lines of code that were executed when the tests were run. If there are lines of code that didn’t run at all, they represent a code path we didn’t test, which could hold potential bugs. Coverage gives you an idea of other tests that are needed. 
+
+## pytest --cov  
+
+To run tests with coverage, run `pytest` and add `--cov` pointing to the module you're covering. There is also an option to generate an html report, which we do in this case. 
+
+    (napari-env) user@directory % 
+    pytest --cov=plugin_tests --cov-report=html .  
+
+Note that the “.” is part of the command.  
+
+    =========================== test session starts ==========================  
+    platform darwin -- Python 3.9.7, pytest-6.2.5, py-1.11.0, pluggy-1.0.0  
+    PyQt 5.15.6 -- Qt runtime 5.15.2 -- Qt compiled 5.15.2  
+    rootdir: qt-4.0.2, napari-plugin-engine-0.2.0, napari-0.4.13, cov-3.0.0  
+    collected 2 items  
+ 
+    src/plugin_tests/_tests/test-reader.py ..                           [100%]  
+ 
+    --------- coverage: platform darwin, python 3.0.7-final-0 --------  
+    Coverage HTML written to dir htmlcov  
+ 
+    =========================== 2 passed in 6.72s ==========================  
+
+This command runs the tests, gets the coverage, and then writes the Coverage HTML report to the htmlcov directory.
+
+There is a large folder (`htmlcov`) in the directory where the tests were run (`plugin_tests`). 
+
+`(napari-env) plugin-tests $ tree -L 1`  
+`.`  
+`├── LICENSE  `  
+`├── MANIFEST.in`  
+`├── README.md`  
+`├── _pycache__`  
+`├── example_func.py`  
+`├── htmlcov		        # <<========================== directory created by pytest-cov`  
```

Comment:
Formatting detail: as this is right now, the line is wrapped in the rendered html page:

![Screenshot_20221123_103848](https://user-images.githubusercontent.com/3949932/203560842-ab187613-0f1e-4f46-a92b-ef9a39439f84.png)

As a general rule, I find that following the 80-character limit for line breaks in this case (especially formatted code) is best. 

```suggestion
`├── htmlcov		        # <<=============== directory created by pytest-cov`
```

You can also mark this whole block as a code block, instead of ticking individual lines.